etc/testing/pipeline-build/go/go.mod

```diff
@@ -1,5 +1,5 @@
 module github.com/pachyderm/pachyderm/etc/testing/pipeline-build/go
 
-go 1.13
+go 1.16
```

Comment:
We haven't updated this in the past, but I don't see any reason not to.

Reply:
I know less about go.mod than you. But reading about it this sets the min version if nothing else has been specified

Reply:
I just meant in the build pipeline.